### DISCUSSION TITLE
TS Error: 'test' does not exist in type 'UserConfigExport'

### DISCUSSION THREAD

**Discussion by michealroberts** [vitest-dev/vitest#292] [Q&A] [answered]
Hi. I'm wondering if someone could advise on this TS error I am seeing. 

When adding the test property into my Vite config file I am seeing the following error:

```
Object literal may only specify known properties, and 'test' does not exist in type 'UserConfigExport'
```

Here is my `vite.config.ts`:

```
import { defineConfig } from 'vite'

import vue from '@vitejs/plugin-vue'
import WindiCSS from 'vite-plugin-windicss'

import typescript from '@rollup/plugin-typescript'

// https://vitejs.dev/config/
export default defineConfig({
  test: {},
  plugins: [
    // https://github.com/antfu/vite-plugin-windicss
    WindiCSS({
      safelist: '',
    }),
    typescript({
        tsconfig: './tsconfig.json'
    }),
    vue()
  ]
})
```

Output of `npx envinfo --system --npmPackages '{vitest,vite,@vitejs/*}' --binaries --browsers`:

```
System:
    OS: macOS 12.0.1
    CPU: (8) arm64 Apple M1
    Memory: 82.53 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.13.0 - /usr/local/bin/node
    npm: 8.1.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 96.0.4664.110
    Firefox: 92.0
    Safari: 15.1
    Safari Technology Preview: 15.4
  npmPackages:
    @vitejs/plugin-vue: ^2.0.1 => 2.0.1 
    vite: ^2.7.6 => 2.7.6 
    vitest: ^0.0.109 => 0.0.109 
```

**Comment by utamori** [comment, marked as answer]
add `/// <reference types="vitest" />`

https://github.com/vitest-dev/vitest/blob/main/test/react/vitest.config.ts

**Reply by ferdicus** [reply]
didn't work for me - is some other additional setup required?

**Reply by Azus5** [reply]
not working for me as well...

**Reply by wmy0320** [reply]
not working for me， too

**Comment by michealroberts** [comment]
@utamori Perfect thank you. I don't see this mentioned in the guide section of the docs ... perhaps a useful addition? :)

**Reply by utamori** [reply]
😃 

It's already been changed by https://github.com/vitest-dev/vitest/commit/adfb67381615d32bfa969f476b4fd5cf0b0b225b, 
but the website doesn't seem to reflect it yet
